index.d.ts

```diff
@@ -0,0 +1,14 @@
+import type { JSONSchema7 } from 'json-schema';
+
+declare module '@asyncapi/specs' {
+  declare const specs: {
```

Comment:
should we have the `declare` here? 

when I tried opening this, I got an error:

```
A 'declare' modifier cannot be used in an already ambient context.ts(1038)
```

I've not created typescript declarations before, so I'm not really sure what the implications of this are - but removing the inner `declare` made the IDE error go away.  🤷

Reply:
hmm, strange. I wasn't getting that error, but I removed that keyword `declare` before `const`. On the stackoverflow people write that since some version of TS the word `declare` is not needed when we declare it inside a module.

Thanks!